pipeline-notebooks/pipeline-section.ipynb

```diff
@@ -62,24 +62,7 @@
    "execution_count": null,
    "id": "be77e966",
    "metadata": {},
-   "outputs": [
```

Comment:
@qued - Did this mess up your toggle switch stuff at all? I didn't see the `<script>` cell, but wasn't sure if I accidentally deleted it or something.

Reply:
Nevermind, saw there's the separate PR open for that still